app/utils/screenshots.py

```
@@ -20,6 +20,7 @@
 from werkzeug.utils import secure_filename
 import urllib3
 from dateutil import tz
+import random
```

Pylint (3.3.7)

[convention] standard import "random" should be placed before third party imports "nodriver", "psutil", "werkzeug.utils.secure_filename", "urllib3", "dateutil.tz"

(C0411)